bin/uglifyjs

```diff
@@ -4,7 +4,7 @@
 
 "use strict";
 
-require("../tools/exit.js");
+// require("../tools/exit.js");
```

Comment:
I had to remove this to get the tests that expect errors to work. I can't reason about this code or how to augment it - ideas?